src/lib/services/export-import-service.ts

```diff
@@ -121,6 +124,7 @@ export default class ExportImportService {
             this.segmentStore.getAllFeatureStrategySegments(),
             this.contextFieldStore.getAll(),
             this.featureTagStore.getAll(),
+            this.segmentStore.getAll(),
```

Comment:
Segments depend on the `featureStrategies`. This means we need to do some` .then() `here, or possibly even better, for future, is to rewrite it entirely in SQL to join features, strategies, segments etc.